hazelcast-spring/src/main/resources/hazelcast-spring-3.10.xsd

```diff
@@ -1774,6 +1791,18 @@
             </xs:appinfo>
         </xs:annotation>
     </xs:element>
+    <xs:element name="cardinalityEstimator" type="hazelcast-type">
+        <xs:annotation>
```

Comment:
this was completely missing in...

Reply:
You can add the `durableExecutorService` and `scheduledExecutorService` as well.

Reply:
Done